docs/overview.md

```diff
@@ -5,90 +5,77 @@ sidebar_label: Overview
-and <a href="https://github.com/openebs/openebs/blob/master/ADOPTERS.md" target="_blank">OpenEBS Adoption stories</a>.
+Some key aspects that make OpenEBS different compared to other traditional storage solutions:
+- Built using the _micro-services architecture_ like the applications it serves. OpenEBS is itself deployed as a set of containers on Kubernetes worker nodes. Uses Kubernetes itself to orchestrate and manage OpenEBS components.
+- Built completely in userspace making it highly portable to run across _any OS/platform_.
+- Completely intent-driven, inheriting the same principles that drive the _ease of use_ with Kubernetes.
+- OpenEBS supports a range of storage engines so that developers can deploy the storage technology appropriate to their application design objectives. Distributed applications like Cassandra can use the _LocalPV_ engine for lowest latency writes. Monolithic applications like MySQL and PostgreSQL can use _Mayastor_ or _cStor based on ZFS_ for resilience. Streaming applications like Kafka can use the NVMe engine [Mayastor](https://github.com/openebs/Mayastor) for best performance in edge environments. Across engine types, OpenEBS provides a consistent framework for high availability, snapshots, clones and manageability.
 
+An added advantage of being a completely Kubernetes native solution is that administrators and developers can interact and manage OpenEBS using all the wonderful tooling that is available for Kubernetes like kubectl, Helm, Prometheus, Grafana, Weave Scope, etc.
 
-<br>
+Check out what users of OpenEBS have to say about their experience in <a href="https://github.com/openebs/openebs/blob/master/ADOPTERS.md" target="_blank">OpenEBS Adoption stories</a>.
 
-## Quickstart
+## Types of OpenEBS Storage Engines
+
+OpenEBS is a collection Storage Engines, allowing you to pick the right storage solution for your Stateful workloads and the type of Kubernetes platform.  At a high-level, OpenEBS supports two broad categories of volumes - Local and Replicated. 
+
+### Local Volumes 
 
-- When using synchronous replication, iSCSI is used to attach storage from OpenEBS to 
-  application pods. Hence OpenEBS requires iSCSI client to be configured and `iscsid` service
-  running on the worker nodes.
-  Verify if [iSCSI service is up](/docs/next/prerequisites.html) and
-  running before starting the installation.
+Local Volumes are accessible only from a single node in the cluster. Pods using Local Volume have to be scheduled on the node where volume is provisioned. Local Volumes are typically preferred for distributed workloads like Cassandra, MongoDB, Elastic, etc that are distributed in nature and have high availiability built into them. 
 
-- Default installation works in most of the cases. As a *Kubernetes cluster-admin*, start the default installation using either
+Depending on the type of storage attached to your Kubernetes worker nodes, you can select from Dynamic Local PV - Hostpath, Device, ZFS or Rawfile.
```

Comment:
you can select from Dynamic Local PV - > you can select different flavors of Dynamic Local PV such as 

It's just a suggestion.